packages/browser-policy-content/browser-policy-content.js

```diff
@@ -56,12 +60,12 @@ BrowserPolicy.content = {};
 var parseCsp = function (csp) {
   var policies = csp.split("; ");
   cspSrcs = {};
-  _.each(policies, function (policy) {
+  policies.forEach(function (policy) {
     if (policy[policy.length - 1] === ";")
       policy = policy.substring(0, policy.length - 1);
     var srcs = policy.split(" ");
     var directive = srcs[0];
-    if (_.indexOf(srcs, keywords.none) !== -1)
+    if (srcs.indexOf(keywords.none) !== -1)
```

Comment:
Maybe we could use includes here. Should be a tiny bit faster as includes returns boolean.